src/AspNetCore/Src/inprocessapplication.cxx

```diff
@@ -46,13 +46,26 @@ http_get_raw_response(
     return pHttpContext->GetResponse()->GetRawHttpResponse();
 }
 
-EXTERN_C __MIDL_DECLSPEC_DLLEXPORT VOID http_set_response_status_code(
+EXTERN_C __MIDL_DECLSPEC_DLLEXPORT HRESULT http_set_response_status_code(
```

Comment:
the signature may need to updated as you may also want to set sub-status 
https://support.microsoft.com/en-us/help/943891/the-http-status-code-in-iis-7-0--iis-7-5--and-iis-8-0

Reply:
This is fixed, diff still showing up.